src/components/interface/NetworkStatusBanner.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div
+    v-if="networkStatus === false"
+    class="fixed mt-20 bg-devils-advocate w-full min-w-screen p-3 pt-6 justify-center float-left text-center text-white sm:flex"
+  >
```

Comment:
@SMITH1103 Can you help me check whether this happens for you too? I'm unsure whether it's an issue on my part or of the banner's

https://user-images.githubusercontent.com/22848332/134089604-c60476e0-265b-4f4c-9e3e-e9d8be61183d.mov

Reply:
@jackthta That was an issue with the screen size (the banner shouldn't move on medium/large screens, only small screens to scroll with the header) -- it should be fixed now.